### DISCUSSION TITLE
Number of wind turbines supported by the super controller

### DISCUSSION THREAD

**Discussion by Hermse** [OpenFAST/openfast#1309] [Q&A]
Hello everyone.
I noticed that FAST.Farm only supports up to nine wind turbines in its super controller. Is there any way to support more wind turbines?I want to do a field level control simulation of a wind plant with more than nine wind turbines.

**Comment by jjonkman** [comment]
Dear @Hermse,

Actually, there is no limit to how many wind turbines can be controlled via the super controller in FAST.Farm.

My guess is you are thinking that because of the limit of 9 turbines in the list of FAST.Farm write outputs (as documented here: https://openfast.readthedocs.io/en/main/source/user/fast.farm/AppendixC.html).  However, this is only an artificial limit to keep α to 1 digit in the naming of the super controller write outputs.  (If you have more than 9 wind turbines, you can only output data from the first 9 via the FAST.Farm `OutList`.)  Within the FAST.Farm super controller, you can write outputs for as many turbines as you want.

Best regards,
